micrometer-core/src/main/java/io/micrometer/core/instrument/Timer.java

```diff
@@ -66,10 +67,10 @@ static Builder builder(Timed timed, String defaultName) {
         }
 
         return new Builder(timed.value().isEmpty() ? defaultName : timed.value())
-                .tags(timed.extraTags())
-                .description(timed.description().isEmpty() ? null : timed.description())
-                .publishPercentileHistogram(timed.histogram())
-                .publishPercentiles(timed.percentiles().length > 0 ? timed.percentiles() : null);
+            .tags(timed.extraTags())
```

Comment:
Seems like here the IDE played its magic and added automatic spaces. I think should be reverted. Similarly in other meters.